lib/article_json/export/apple_news/elements/image.rb

```diff
@@ -3,14 +3,37 @@ module Export
     module AppleNews
       module Elements
         class Image < Base
+          # Image | Image, Caption
+          # @return [Hash] | [Hash], [Hash]
+          def export
+            caption_text.nil? ? image : [image, caption].join(',')
```

Comment:
This looks weird, you're converting the hashes into a string here. Wouldn't it make more sense to return the array of hashes here?
```suggestion
            caption_text.nil? ? image : [image, caption]
```
BTW, the YARD notation for this is would then be `@return [Hash, Array<Hash>]`

Reply:
So, I was trying to return the JSON contents (without the array) but I realise now that I can/should just call ```.flatten``` on the components array.